.github/workflows/ci_tests.yml

```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   preset-test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest # 22.04
```

Comment:
Note: Ubuntu-latest will switch to 24.04 in Jan 2025, see: https://github.com/actions/runner-images/issues/10636

Reply:
and next than you will use ubuntu-22.04?

Reply:
yes. I am trying to point out that this comment will become out of date soon. I do not foresee our pipeline to fail at 24.04.

Reply:
OK